src/Flow/Debounce.cs

```diff
@@ -41,19 +41,24 @@ public static async Task<bool> Debounce(string debouncerKey, TimeSpan timeToWait
                 return false;
             }
 
-            try
+            // credits: https://stackoverflow.com/a/30740088
+            var delayTask = Task.Delay(timeToWait, debouncerData.TokenSource.Token);
+            var continuationTask = delayTask.ContinueWith(task =>
             {
-                await Task.Delay(timeToWait, debouncerData.TokenSource.Token);
-                Cleanup();
+                // Don't throw the exception if the task cancellation was requested Transfer the insidie Inside delay.Task.IsCancelled.
```

Comment:
-2, typos, variable name in comment. Incorrect variable name as well.
(no penalty) Reworded this to read a bit easier.

The reason we shouldn't include variable names here is because some IDEs (notably, vanilla VS) won't rename variable names in comments, causing it to get out of date with the codebase when things get refactored or renamed, which is what appears to have happened here.

```suggestion
                // Do nothing. Silences task cancellation exceptions. See https://stackoverflow.com/a/30740088
```